test/expects/pug.js

```diff
@@ -1,3 +1,3 @@
-var pug = { template: "<div class=\"pug__test keep-me\"><article><p>foo</p></article></div>",cssModules: {"test":"pug__test"},};
+var pug = { template: "<div class=\"pug__test keep-me\" v-if=\"true\"><article><p>foo</p></article></div><p v-else=\"v-else\">nothing</p>",cssModules: {"test":"pug__test"},};
```

Comment:
I think we need specify `doctype` option for pug compiler.
https://pugjs.org/language/doctype.html#doctype-option